packages/app-extensions/src/notification/modules/center/NotificationCenter/NotificationCenter.js

```diff
@@ -26,39 +29,44 @@ const NotificationCenter = (
   }, [])
 
   const handleScroll = () => {
-    if (element.current.scrollTop + element.current.offsetHeight >= element.current.scrollHeight) {
-      if (!isLoadingMoreNotifications && moreNotificationsAvailable) {
-        loadNotifications(Object.keys(notifications).length)
-      }
+    const notificationCount = Object.keys(notifications).length
+    const isAtBottom = element.current.scrollTop + element.current.offsetHeight >= element.current.scrollHeight
+    const canLoad = !isLoadingMoreNotifications && moreNotificationsAvailable
+
+    if (isAtBottom && canLoad) {
+      loadNotifications(notificationCount)
     }
   }
 
-  const sortedNotifications = Object.keys(notifications)
+  const SortedNotifications = Object.keys(notifications)
     .map(k => notifications[k])
-    .filter(n => !n.result || n.result.type !== resultTypes.outputjob || n.result.file)
-    .filter(n => !n.taskProgress || n.taskProgress.status !== 'cancelled')
+    .filter(n =>
```

Comment:
I think there should be two filters. for example if _n.result.type === resultTypes.entities_ and _n.taskProgress.status === 'cancelled'_ in the old one the element is filtered out. With only one filter the notification is still in the list (which is wrong)